src/components/BreadcrumbView/BreadcrumbView.tsx

```diff
@@ -97,37 +97,39 @@ export class BreadcrumbView extends React.Component<BreadCumbViewProps, BreadCum
     const { namespace, itemName, item, istioType, pathItem } = this.state;
     const isIstio = this.isIstio();
     const linkItem = isIstio ? (
-      <Breadcrumb.Item componentClass="span" active={true}>
+      <BreadcrumbItem isActive={true}>
         {itemName}: {item}
-      </Breadcrumb.Item>
+      </BreadcrumbItem>
     ) : (
-      <Breadcrumb.Item componentClass="span">
+      <BreadcrumbItem isActive={true}>
         <Link to={this.getItemPage()} onClick={this.cleanFilters}>
           {itemName}: {item}
         </Link>
-      </Breadcrumb.Item>
+      </BreadcrumbItem>
     );
     return (
-      <Breadcrumb title={true}>
-        <Breadcrumb.Item componentClass="span">
-          <Link to={`/${pathItem}`} onClick={this.cleanFilters}>
-            {isIstio ? IstioName : BreadcrumbView.capitalize(pathItem)}
-          </Link>
-        </Breadcrumb.Item>
-        <Breadcrumb.Item componentClass="span">
-          <Link to={`/${pathItem}?namespaces=${namespace}`} onClick={this.cleanFilters}>
-            Namespace: {namespace}
-          </Link>
-        </Breadcrumb.Item>
-        {isIstio && (
-          <Breadcrumb.Item componentClass="span">
-            <Link to={`/${pathItem}?namespaces=${namespace}`} onClick={this.updateTypeFilter}>
-              {itemName} Type: {istioType}
+      <div className="breadcrumb">
+        <Breadcrumb>
```

Comment:
Is there any style missing for this div ?

Reply:
This is not part of the PR anymore. Breadcrumb was moved in a previous PR. Outdated code, sorry.